### PR TITLE
Fix version inconsistencies in remaining files

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
Sub-PR of #3368 for the fixes of the files themselves.

### CODE REVIEW

File: css/properties/inline-size.json [hunk 1]
```diff
@@ -196,8 +196,7 @@
                 "version_added": false
               },
               "firefox": {
-                "prefix": "-moz-",
-                "version_added": "3"
+                "version_added": "38"
```
Comment (Elchi3) [inline css/properties/inline-size.json]: I don't understand why this would be 38. Can you explain?

File: css/properties/max-height.json [hunk 2]
```diff
@@ -161,11 +161,11 @@
             "support": {
               "chrome": {
                 "alternative_name": "-webkit-fill-available",
-                "version_added": "28"
+                "version_added": "25"
```
Comment (Elchi3) [inline css/properties/max-height.json]: I don't understand these changes to this entire file. Can you explain what was inconsistent and how you resolved it?

File: css/properties/inline-size.json (hunk 1 above)
Comment (vinyldarkscratch) [reply css/properties/inline-size.json]: Both of your comments are related to one another.  All of the css.[min/max][height/block-size/inline-size] features were introduced to the W3 spec at the same time, and based upon the data I've found from CanIUse and Firefox's commits (which I totally should have saved for this exact reason), it didn't seem like any of these properties were implemented before any of their siblings.

Comment (vinyldarkscratch) [reply css/properties/inline-size.json]: I'm reverting and tweaking these changes in favor of working alongside #3624.

**Pull request merged**